src/ImageSharp.Web/Caching/SHA256CacheHash.cs

```diff
@@ -33,18 +33,16 @@ public SHA256CacheHash(IOptions<ImageSharpMiddlewareOptions> options)
         public string Create(string value, uint length)
         {
             int byteCount = Encoding.ASCII.GetByteCount(value);
-
-            // Allocating a buffer from the pool is ~27% slower than stackalloc so use that for short strings
-            if (byteCount < 257)
-            {
-                return HashValue(value, length, stackalloc byte[byteCount]);
-            }
-
             byte[] buffer = null;
+
             try
             {
-                buffer = ArrayPool<byte>.Shared.Rent(byteCount);
-                return HashValue(value, length, buffer.AsSpan(0, byteCount));
+                // Allocating a buffer from the pool is ~27% slower than stackalloc so use that for short strings
+                Span<byte> bytes = byteCount <= 128
+                    ? stackalloc byte[byteCount]
```

Comment:
There was a Twitter thread recently about using static value for stackalloc being better, like stackalloc[128].AsSpan(requested).

https://twitter.com/EgorBo/status/1508069816275513344?t=zxpY4AQ_9JddgbLKat0Yrw&s=19

Reply:
Ah yeah, then you slice it. I remember that Tweet. Thanks!

Reply:
Cool,  I can't remember if SkipLocalsInit could  be just shim for older frameworks via custom attribute and compiler would pick it up. Great work here as always!

Reply:
No idea tbh! It's really hard to keep track of what is shimmable. Thanks!